src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioSymbolNavigationService.cs

```diff
@@ -50,7 +49,6 @@ internal partial class VisualStudioSymbolNavigationService : ForegroundThreadAff
             var componentModel = IServiceProviderExtensions.GetService<SComponentModel, IComponentModel>(_serviceProvider);
             _editorAdaptersFactory = componentModel.GetService<IVsEditorAdaptersFactoryService>();
             _metadataAsSourceFileService = componentModel.GetService<IMetadataAsSourceFileService>();
-            _sourceGeneratedFileManager = componentModel.GetService<SourceGeneratedFileManager>();
```

Comment:
📝 All of these should be promoted to constructor parameters so MEF can create the objects prior to started to construct this object.

Reply:
@sharwell This constructor isn't marked as `ImportingConstructor`.